eclass/qmake-utils.eclass

```diff
@@ -1,71 +1,75 @@
-# Copyright 1999-2019 Gentoo Authors
+# Copyright 1999-2020 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
 # @ECLASS: qmake-utils.eclass
 # @MAINTAINER:
 # qt@gentoo.org
 # @AUTHOR:
 # Davide Pesavento <pesa@gentoo.org>
-# @SUPPORTED_EAPIS: 6 7
+# @SUPPORTED_EAPIS: 7
 # @BLURB: Common functions for qmake-based packages.
 # @DESCRIPTION:
-# Utility eclass providing wrapper functions for Qt4 and Qt5 qmake.
+# Utility eclass providing wrapper functions for Qt5 qmake.
 #
 # This eclass does not set any metadata variables nor export any phase
 # functions. It can be inherited safely.
 
 if [[ -z ${_QMAKE_UTILS_ECLASS} ]]; then
 _QMAKE_UTILS_ECLASS=1
 
-[[ ${EAPI:-0} == [012345] ]] && die "qmake-utils.eclass: unsupported EAPI=${EAPI:-0}"
+case ${EAPI} in
+	7) ;;
+	*) die "EAPI=${EAPI:-0} is not supported" ;;
+esac
 
 inherit estack toolchain-funcs
 
+# @FUNCTION: _qmake-utils_banned_func
+# @INTERNAL
+# @DESCRIPTION:
+# Banned functions are banned.
+_qmake-utils_banned_func() {
+	die "${FUNCNAME[1]} is banned in EAPI 7 and later"
```

Comment:
[very minor, feel free to ignore] "in EAPI 7 and later" may be misleading... does it imply it's _not_ banned in EAPI 6 and earlier?

Reply:
right, it's just a continuation of the existing message, when one earlier EAPI had still existed.